app/templates/_package.json

```diff
@@ -24,24 +24,18 @@
   },
   "dependencies": {},
   "devDependencies": {
-    "babel-cli": "^6.24.0",
-    "babel-core": "^6.24.0",
-    "babel-plugin-transform-object-rest-spread": "^6.23.0",
+    "babel-cli": "^6.26.0",
+    "babel-core": "^6.26.0",
+    "babel-plugin-transform-object-rest-spread": "^6.26.0",
     "babel-plugin-transform-flow-strip-types": "^6.22.0",
-    "babel-preset-env": "^1.2.2",
+    "babel-preset-env": "^1.6.0",
     "babel-eslint": "^7.2.1",
-    "eslint": "^3.18.0",
-    "eslint-config-airbnb": "^14.1.0",
-    "eslint-config-prettier": "^1.5.0",
-    "eslint-plugin-flowtype": "^2.30.4",
-    "eslint-plugin-import": "^2.2.0",
-    "eslint-plugin-jsx-a11y": "^4.0.0",
-    "eslint-plugin-prettier": "^2.0.1",
-    "eslint-plugin-react": "^6.10.3",
-    "flow-bin": "^0.42.0",
-    "jest": "^19.0.2",
+    "eslint": "^4.6.1",
+    "eslint-config-callstack-io": "^0.4.1",
+    "flow-bin": "^0.54.1",
+    "jest": "^21.0.2",
     "npm-release": "^1.0.0",
-    "prettier": "^0.22.0"
+    "prettier": "^1.6.1"
```

Comment:
I guess since we run `eslint` and `eslint --fix`, we don't need to directly depend on `prettier` anymore.